cfgov/v1/tests/wagtail_pages/test_page_titles.py

```diff
@@ -0,0 +1,99 @@
+from django.test import TestCase
+from django.test import Client
+
+from scripts import _atomic_helpers as atomic
+from v1.models.landing_page import LandingPage
+from v1.models.sublanding_page import SublandingPage
+from v1.models.browse_page import BrowsePage
+from v1.models.browse_filterable_page import BrowseFilterablePage, EventArchivePage, NewsroomLandingPage
+from v1.models.sublanding_filterable_page import SublandingFilterablePage, ActivityLogPage
+from v1.models.learn_page import EventPage, LearnPage, DocumentDetailPage
+from v1.models.newsroom_page import NewsroomPage, LegacyNewsroomPage
+from v1.models.blog_page import BlogPage, LegacyBlogPage
+
+
+from helpers import publish_page
+
+
+'''
+Page types tested here: 
+
+LandingPage,
+SubLandingPage,
+BrowsePage,
+BrowseFilterablePage,
+SublandingFilterablePage,
+EventArchivePage,
+DocumentDetailPage,
+LearnPage,
+EventPage,
+NewsroomLandingPage,
+NewsroomPage,
+LegacyNewsroomPage,
+BlogPage,
+LegacyBlogPage,
+ActivityLogPage
+ 
+'''
+
+django_client = Client()
+
+class PageTitlesTestCase(TestCase):
+    """ Tests that all Wagtail page types load and display the title correctly """
+
+    def page_loads_with_correct_title(self, page_type):
+        publish_page(
+            globals()[page_type](
+                title = 'Title ABCD',
+                slug = 'page',
+            )
+        )
+        response = django_client.get('/page/')
+        self.assertContains(response, 'Title ABCD')
+
+    def test_sublanding_page(self):
+        self.page_loads_with_correct_title(page_type = 'SublandingPage')
```

Comment:
There's also a few other places where the PR isn't compliant with PEP8, btw.   There's a couple of ways, I think, that you guys could decide upon if you want to standardize the code formatting.

1)  pip has a module for checking pep8 compliance: https://pypi.python.org/pypi/pep8

2) put the settings in your IDE.  if you use sublime, you can use something like this: https://packagecontrol.io/packages/Python%20PEP8%20Autoformat, and it'll automatically point out wherever there's some PEP8 issue.  If you use PyCharm, that feature should be built in.